go.mod

```diff
@@ -24,6 +24,7 @@ replace (
 	github.com/Azure/go-autorest v11.1.2+incompatible => github.com/Azure/go-autorest/autorest v0.10.0
 	github.com/dgrijalva/jwt-go => github.com/golang-jwt/jwt/v4 v4.1.0
 	github.com/gogo/protobuf => github.com/gogo/protobuf v1.3.2
+	github.com/microsoft/moc => ../moc
```

Comment:
Could you fix the path here to point to your change in moc repo on github. This doc describes how to do it.

https://msazure.visualstudio.com/msk8s/_wiki/wikis/msk8s.wiki/646685/Testing-dependent-PRs

Reply:
Sure